### ISSUE TITLE
Flag a crop for edit or merging

### ISSUE THREAD

**Issue by CloCkWeRX** [Growstuff/growstuff#1046]
We've got:
 * http://www.growstuff.org/crops/habanero-pepper
 * http://www.growstuff.org/crops/habanero

both with content. As a non crop wrangler, there's nothing I can do to fix this - can't flag a problem or recommend a merge.

Though we can add alternative names, there's no current facility to mark one crop as an alias of or superceded by another either; nor to move content.

I think marking it as superceded is the better option, as that way we don't have to migrate blog post links, plantings, photos, etc.

**Comment by pozorvlak** [comment]
Agreed that this would be a good feature to have in general. I'd suggest
the following subtasks:

 - allow users to mark crops as needing crop-wrangler attention, with a
comment as to why;
 - allow crop wranglers to mark crops as superseded;
 - write code to merge crops (migrate blog posts, etc - we need to think
about an "undo" feature here);
 - run a one-off "merge all superseded crops" task;
 - replace the "mark superseded" button with "merge".

In this specific case, "habanero pepper" has no plantings and one post; we
could edit the post to refer to "habanero" and then delete "habanero
pepper". Does that sound OK?

Also, @CloCkWeRX, would you like to be made a crop wrangler?

**Comment by CloCkWeRX** [comment]
>
>
> Also, @CloCkWeRX, would you like to be made a crop wrangler?
>

Yeah, why not :)

**Comment by maco** [comment]
> - write code to merge crops (migrate blog posts, etc - we need to think
about an "undo" feature here);

that sounds like having superceding blog post versions
